src/inc/corinfo.h

```diff
@@ -276,12 +276,26 @@ enum SystemVClassificationType : unsigned __int8
     SystemVClassificationTypeMemory             = 3,
     SystemVClassificationTypeInteger            = 4,
     SystemVClassificationTypeIntegerReference   = 5,
-    SystemVClassificationTypeSSE                = 6,
+    SystemVClassificationTypeIntegerByRef       = 6,
+    SystemVClassificationTypeSSE                = 7,
     // SystemVClassificationTypeSSEUp           = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeX87             = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeX87Up           = Unused, // Not supported by the CLR.
     // SystemVClassificationTypeComplexX87      = Unused, // Not supported by the CLR.
-    SystemVClassificationTypeMAX = 7,
+
+    // Internal flags - never returned outside of the classification implementation.
```

Comment:
No. The classification returns to through the Jit interface only the integral types and the SSE type. This enum value is used internally by the classification algorithm - it is a counterpart to the VM fields' ELEMENT_TYPE_TYPEDBYREF.